Fix issue for working on windows

imjlk opened the pull request:
#78 
I'm running on Windows environment with Docker desktop for Windows (WSL2).

luangjokaj commented:
Tested and works with Windows Subsystem for Linux ✅

ribaricplusplus commented:
I think this is actually might be pointless and just causes stuff like https://github.com/luangjokaj/wordpressify/discussions/109

imjlk commented:
> I think this actually might be pointless and just causes stuff like #109
> 
> There is no point in calling `id -u` on Windows. It doesn't fix the related issue. It just makes a different error show up for Windows (non WSL), which people don't find in open WordPressify issues and so open a new one (like the discussion above). Also complicates code unnecessarily.
> 
> I would advise reverting this PR.

I'm not sure what happened on #109, but my PR is working for Windows environment.

NodeJS for Windows, there is no method process.getuid & getgid.
> https://nodejs.org/dist/latest-v14.x/docs/api/process.html#process_process_getuid
So It cause error when it replace WPFY_UID & WPFY_GID from .env.in and .dockerfile.in

Maybe my works is not proper, I just referenced like these
[[fix: removed process.getuid method as its not supported in winos by anikethsaha · Pull Request #966 · webpack/webpack-cli]](https://github.com/webpack/webpack-cli/pull/966)

And I looked for a way to check UID and GID in the Windows environment, and just applied it right away because execSync was already imported. That's why i added `id -u` and `id -g`

Thank you for your appreciation.

ribaricplusplus commented:
Correct me if I'm wrong, but I don't think Windows has a concept of "user id". If you open the command prompt in Windows and run `id -u`, it's not going to work.

If it were possible to get a UID on Windows, then Node.js team would make it happen by running `id -u` themselves.

imjlk commented:
Actually I don't know much about this in detail so I can't explain it. On my Windows machine, `id -u` and `id -g` return same Integer on terminal.

ribaricplusplus commented:
> Oh it's not working on cmd

Yep
